rj_gameplay/rj_gameplay/action/pivot.py

```diff
@@ -31,9 +31,9 @@ def tick(self, intent: RobotIntent) -> None:
 
     def is_done(self, world_state:rc.WorldState) -> bool:
         #TODO: Change this when we get action state feedback
-        angle_threshold = 0.1
+        angle_threshold = np.pi / 180 #One dgree
```

Comment:
```suggestion
        angle_threshold = np.pi / 180 #One degree
```
Small typo in a useful comment